irods/manager/user_manager.py

```diff
@@ -27,6 +28,25 @@ def get(self, user_name, user_zone=""):
             raise UserDoesNotExist()
         return iRODSUser(self, result)
 
+    # pending resolution of https://github.com/irods/irods/issues/6887
```

Comment:
Please somehow link this issue to the issue to which this change is related (and/or to this PR) in GitHub for future investigation

Reply:
Done